### ISSUE TITLE
org.restlet.engine.io.UnclosableOutputStream cannot be cast to java.lang.Appendable

### ISSUE THREAD

**Issue by dgrant** [immutables/immutables#571]
Version:
2.4.4-SNAPSHOT (githash: c44caea1eb2403b9a6e93f81591624a5bb2d4deb)

Stacktrace:
```
java.lang.ClassCastException: org.restlet.engine.io.UnclosableOutputStream cannot be cast to java.lang.Appendable
	at org.immutables.gson.stream.GsonMessageBodyProvider$1$1.write(GsonMessageBodyProvider.java:483)
	at org.immutables.gson.stream.GsonMessageBodyProvider.writeTo(GsonMessageBodyProvider.java:145)
	at org.restlet.ext.jaxrs.internal.wrappers.provider.SingletonProvider.writeTo(SingletonProvider.java:450)
	at org.restlet.ext.jaxrs.internal.util.JaxRsOutputRepresentation.write(JaxRsOutputRepresentation.java:104)
	at org.restlet.engine.application.EncodeRepresentation.write(EncodeRepresentation.java:295)
	at org.restlet.engine.adapter.ServerCall.writeResponseBody(ServerCall.java:513)
	at org.restlet.engine.adapter.ServerCall.sendResponse(ServerCall.java:457)
	at org.restlet.ext.servlet.internal.ServletCall.sendResponse(ServletCall.java:453)
	at org.restlet.engine.adapter.ServerAdapter.commit(ServerAdapter.java:191)
	at org.restlet.engine.adapter.HttpServerHelper.handle(HttpServerHelper.java:144)
	at org.restlet.ext.servlet.ServerServlet.service(ServerServlet.java:1117)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.mortbay.jetty.servlet.ServletHolder.handle(ServletHolder.java:511)
	at org.mortbay.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1166)
```

**Comment by elucash** [comment]
released in 2.4.4. thank you!

**Comment by dgrant** [comment]
Thank you!
